alerts.css

```diff
@@ -49,38 +27,18 @@
     padding:1rem;
 }
 .greenalert{
-<<<<<<< HEAD
-<<<<<<< HEAD
-    background-color: #77D970;
+    background-color: var(--greenalertCol);
 }
 
-.bluealert{
-    background-color: #3DB2FF;
-}
-.yellowalert{
-    background-color: #F9D371;
-}
 
-/* badges */
 .badges{
     color:rgb(202, 186, 186);
     display: flex;
-    
-
-=======
-=======
-
->>>>>>> 34241759428d49fb2873f73b4e4ae05c401a9197
-    background-color: var(--greenalertCol);
-}
+    }
 
 .bluealert{
     background-color: var(--bluealertCol);
 }
 .yellowalert{
     background-color:var(--yellowalertCol) ;
```

Comment:
just a suggestion give color naming with " - " or use camel casing it's hard to red